diff --git a/docusaurus.config.js b/docusaurus.config.js
index 86eed89..64a600b 100644
--- a/docusaurus.config.js
+++ b/docusaurus.config.js
@@ -114,6 +114,12 @@ const config = {
             label: 'Reference',
             position: 'right',
           },
+          {
+            type: 'doc',
+            docId: 'tutorials/intro',
+            label: 'Tutorials',
+            position: 'right',
+          },
           {
             href: 'https://github.com/wandb/wandb',
             label: 'GitHub',
diff --git a/sidebars.js b/sidebars.js
index 0bb0ce1..5c705ad 100644
--- a/sidebars.js
+++ b/sidebars.js
@@ -345,6 +345,9 @@ const sidebars = {
       dirName: 'ref',
     },
   ],
+  tutorials: [
+    'tutorials/quickstart.qmd'
+  ],
 };
 
 module.exports = sidebars;
